### PR TITLE
Updated to Swift 4.0.

### PR DESCRIPTION
**Pull request by elland**

### CODE REVIEW

File: Sources/Data+Sweetness.swift [hunk 1]
```diff
@@ -3,21 +3,22 @@ import Foundation
...
+        
         if base64.hasSuffix("==") {
-            return base64.substring(to: base64.index(base64.endIndex, offsetBy: -2))
+            return String(base64[base64.startIndex..<base64.index(base64.endIndex, offsetBy: -2)])
         } else if base64.hasSuffix("=") {
-            return base64.substring(to: base64.index(base64.endIndex, offsetBy: -1))
+            return String(base64[base64.startIndex..<base64.index(base64.endIndex, offsetBy: -1)])
         }
-
+
```
Comment (3lvis) [inline Sources/Data+Sweetness.swift]: Could clean up these added spaces? There are a few in this PR

**Review by 3lvis — CHANGES_REQUESTED** [review]
Great initiative!

**Comment by elland** [comment]
👍

**Review by 3lvis — APPROVED** [review]

**Comment by 3lvis** [comment]
giphy 3

**Pull request merged**